core-services/prow/02_config/operator-framework/operator-marketplace/_pluginconfig.yaml

```diff
@@ -17,7 +17,7 @@ label:
       - KeenonLee
       label: cherry-pick-approved
     - allowed_teams:
-      - openshift-patch-managers
+      - openshift-staff-engineers
```

Comment:
before adding this, i'd like to know why this didn't get you the permission:
https://github.com/openshift/release/blob/bc2618b19cb79278ca12c02c6eb59d51cd2e804e/core-services/prow/02_config/_plugins.yaml#L642-L646

i don't think you should need this at a repo level.

Reply:
(specifically because https://github.com/openshift/release/blob/bc2618b19cb79278ca12c02c6eb59d51cd2e804e/core-services/prow/02_config/_plugins.yaml#L642-L646 should already be getting you this permission)

i wonder if it's an org issue (openshift org vs operator-framework org)